lib/encoder_unittest.py

```diff
@@ -214,6 +221,20 @@ def test_OtherFormatter(self):
       formatter=encoder.OptionFormatter(prefix='-', infix=' '))
     self.assertEqual('-foo foo -xyz abc', valueset.ToString())
 
+  def test_RandomlyRemoveParameterSuccessfully(self):
+    config = encoder.OptionValueSet(
+      encoder.OptionSet(encoder.Option('foo', ['foo', 'bar'])),
```

Comment:
Nit: should indent 4 since you're breaking after a left paren. This file appears to do it wrong in more places.